src/components/text-editor/text-editor.jsx

```diff
@@ -35,18 +39,27 @@ class TextEditor extends React.Component {
 
     render () {
         return (
-            <MonacoEditor
-                height="600"
-                language="javascript"
-                theme="vs-dark"
-                width="800"
-            />
+            <div className={styles.editorContainer}>
+                <MonacoEditor
+                    height="100%"
+                    language="javascript"
+                    options={options}
+                    theme="vs-dark"
+                    value={hello}
+                    width="100%"
+                />
+            </div>
         );
     }
 }
 
 TextEditor.propTypes = {
```

Comment:
Is this leftover code? I know that the blocks was previously being used, but I don't think we have it anymore. You can go ahead and get rid of this for your pull request.